pegasus/sites.v3/code.org/views/about_donors_image.haml

```diff
@@ -11,18 +11,24 @@
     text-align: center;
   }
 
+// Size large is a 2 column grid with larger height for larger logos
+// Size medium is a 3 column grid with medium height for logos
+// Size small is a 3 column grid for small logos or name only
+- columns = size == "large" ? 2 : 3
+- height = size == "large" ? 150 : size == "medium" ? 200 : 100
+- fontsize = size == "small" ? "18" : "14"
+
 %table
-  - supporters.each_slice(columns).with_index do |group, groupindex|
+  - supporters.each_slice(columns) do |group|
```

Comment:
do we actually need to know how many items per row, or could we just have them all next to each other and let the widths do the wrapping?